backend/infrahub/graphql/types/task.py

```diff
@@ -28,9 +28,21 @@ class Task(ObjectType):
     start_time = String(required=False)
 
 
+class TaskRelatedNode(ObjectType):
+    id = String(required=False)
+    kind = String(required=False)
```

Comment:
Shouldn't both of these be required=True? I.e. either we'd have an empty list or the entries of that list would have values for both of these if requested?

Reply:
good point